README.rst

```diff
@@ -1758,15 +1758,22 @@ the query were written in the opposite order:
 SQL
 ---
 
-Currently we only support filtering and outputting properties of a single vertex/table. Support
-for edge traversals and most directives is in development.
+Relational databases are supported by compiling to SQLAlchemy core as an intermediate
+language, and then relying on SQLAlchemy's compilation of the dialect specific SQL string to query
+the target database.
+
+Our SQL backend supports basic traversals, filters, tags and outputs, but there's still some pieces
+in development:
+- Directives: `@optional`, `@fold`, `@recurse`
```

Comment:
This will not render correctly in rst. All lists must be preceded with a blank line. Additionally, also please change `@optional` to :code:`optional` and similarly for other highlighted code blocks.